model-optimizer/mo/utils/versions_checker.py

```diff
@@ -60,31 +60,45 @@ def parse_and_filter_versions_list(required_fw_versions, version_list, env_setup
 
     # check environment marker
     if len(splited_requirement) > 1:
```

Comment:
are you sure that markers are semi-column separated? Please take a look at this where it seems it can be boolean expression: https://www.python.org/dev/peps/pep-0496/

Reply:
Resolve as we discussed at meeting - we skip requirement with more then one markers and show a message in log. Tests were update to cover this scenario.